### PR TITLE
no bug - fix config key, should be username not password here

### PR DESCRIPTION
**Pull request by rhelmer**
r? @peterbe

### CODE REVIEW

File: webapp-django/crashstats/settings/base.py [hunk 1]
```diff
@@ -555,7 +555,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
                 'aPassword'
             ),
             'database_username': config(
-                'DATASERVICE_DATABASE_PASSWORD',
+                'DATASERVICE_USERNAME',
```
Comment (peterbe) [inline webapp-django/crashstats/settings/base.py]: Shouldn't that be `DATASERVICE_DATABASE_USERNAME` to follow suit?

**Comment by peterbe** [comment]
Would you mind perhaps tying this to a bug and include one more fix? [This](https://github.com/rhelmer/socorro/blob/nobug-fix-config-key/webapp-django/crashstats/settings/base.py#L569) should be `database_port`.

**Comment by peterbe** [comment]
The reason for tying it to a bug is because I think yanking the `local.py-dist` is quite a landmark move.

**Comment by peterbe** [comment]
If you merge and rebase in https://github.com/rhelmer/socorro/pull/23 this should be good to go.

**Comment by rhelmer** [comment]
@peterbe hah! I didn't actually mean to `rm` the `local.py-dist` in this PR, just fix that one typo! I've reverted that bit.

**Comment by rhelmer** [comment]
@peterbe ok how's that

**Comment by peterbe** [comment]
r+

**Pull request merged**